src/utils/group_helpers.py

```
@@ -4,17 +4,27 @@
+    id: int
+    members: list[dict[str, Any]]
+    count: int
+    averages: dict[str, float]
+    sums: dict[str, float]
+
+
 def aggregate_groups(
     df: pd.DataFrame,
     col_group: str,
     score_cols: list[str],
     col_name: str,
```

Ruff (0.15.12)

[warning] Unused function argument: `col_name`

(ARG001)